app/controllers/application.js

```diff
@@ -12,10 +12,15 @@ const {
 
 export default Controller.extend({
   smt: service(),
+  storage: service('remotestorage'),
   spaces: alias('smt.spaces'),
 
+  // Make the RS instance available in the browser console for development
+  setRemoteStorageGlobal: function() {
+    window.remoteStorage = this.get('storage.rs');
+  }.on('init'),
```

Comment:
Oh, right. For that it actually needs to be an instance initializer in `app/instance-initializers/` instead. The you can do something like:

```
initialize(instance) {
  let storage = instance.lookup('service:remotestorage');
  ...
```